### PR TITLE
Fix/loader simplification

### PR DESCRIPTION
**Pull request by Marcin-Malek**
I simplified Loader component to eliminate browsers compatibility concerns. I also noticed that layout and tiles with people are different when searching and fixed that too. Since person tile wont be wider than 500px i changed images size to 500x750 instead of 780x1180.

### CODE REVIEW

File: src/common/PersonTile/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const PersonTile = ({name, role, portrait, id}) => {
 
     return (
         <Content onClick={() => navigate(`../people/${id}`)}>
-            <Img src={portrait === "https://image.tmdb.org/t/p/w780/null" ? null : portrait} alt={name} loading="lazy" />
+            <Img src={portrait === "https://image.tmdb.org/t/p/w500/null" ? null : portrait} alt={name} loading="lazy" />
```
Comment (damianmichalowski) [inline src/common/PersonTile/index.js:9]: why changing it to 500 ?

File: src/common/PersonTile/styled.js [hunk 2]
```diff
@@ -19,6 +19,10 @@ export const Content = styled.li`
     &:active {
         transform: scale(1.03);
     }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        padding: 8px;
```
Comment (damianmichalowski) [inline src/common/PersonTile/styled.js:24]: good you saw that it is missing

**Review by damianmichalowski — APPROVED** [review]
it is nice you change wrappers in movies and people to moviewrapper and people wrapper and nice you change it, now we have less code, spinner look great, great job

File: src/common/PersonTile/index.js (hunk 1 above)
Comment (Marcin-Malek) [reply src/common/PersonTile/index.js:9]: It loads faster, we dont need w780, it is too big and not changes much.

**Review by JamieShifter — APPROVED** [review]

**Pull request merged**